csharp/tests/Integration/TestConfiguration.cs

```diff
@@ -39,33 +56,40 @@ public IntegrationTestBase(IMessageSink diagnosticMessageSink)
         StopServer(false);
 
         // Delete dirs if stop failed due to https://github.com/valkey-io/valkey-glide/issues/849
-        Directory.Delete(Path.Combine(_scriptDir, "clusters"), true);
+        string clusterLogsDir = Path.Combine(_scriptDir, "clusters");
+        try
+        {
+            Directory.Delete(clusterLogsDir, true);
+        }
+        catch (DirectoryNotFoundException) { }
```

Comment:
```suggestion
        if (Directory.Exists(clusterLogsDir))
        {
            Directory.Delete(clusterLogsDir, true);
        }
```
We should avoid using try...catch if it is possible to detect the error condition up front and execute code accordingly.

Reply:
That dir isn't part of the repo, it does not exist on the first IT run.

Reply:
Right.. So a `Directory.Exists()` check will ensure we only delete the directory if it actually exists.